client/src/partials/Navbar.tsx

```diff
@@ -22,28 +23,37 @@ const StyledNavbar = styled.div`
   color: ${(props) => props.theme.colors.black};
   font-size: ${(props) => props.theme.fontSizes.small};
   font-family: ${(props) => props.theme.fonts.heading};
-  width: 100%;
   z-index: 1;
   position: -webkit-sticky;
   position: sticky;
   top: 0;
-  padding: 20px;
+  padding: 20px 100px;
+  margin: auto;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+`;
+
+const NavbarLogo = styled.div`
+  color: ${(props) => props.theme.colors.insignificant};
+  font-weight: bold;
 `;
 
 const StyledNavbarList = styled.div`
   display: flex;
   flex-direction: row;
   gap: 3vw;
-  margin-right: 50px;
 `;
 
 const StyledNavbarLink = styled.div`
   a {
     text-decoration: none;
     color: ${(props) => props.theme.colors.insignificant};
+    transition: 0.5s;
+    font-weight: bold;
   }
   a:hover {
-    color: #ffffff;
+    color: ${(props) => props.theme.colors.moderate};
```

Comment:
Can we change to teal or pink - I don't find the yellow fits with our colour scheme as well

Reply:
i went with teal. the pink is too bright. though if we wanna change it after its an ez change. we just need to sit down and unanimously agree on the color scheme